opencsp/app/sofast/lib/CalibrateDisplayShape.py

```diff
@@ -468,6 +487,53 @@ def format_image(axis: plt.Axes, im):
         im.set_clim(-self.xyz_screen_map_clim_mm, self.xyz_screen_map_clim_mm)
         ax.set_title('Z (mm)')
 
+    def visualize_unwrapped_phase(self) -> None:
+        """Visualizes x/y unwrapped phase for all poses"""
+        for idx_pose in range(self.data_calculation.num_poses):
+            # Get phase/mask data
+            im_x = self.data_input.measurements_screen[idx_pose].mask_images[..., 1].copy()
+            im_y = self.data_input.measurements_screen[idx_pose].mask_images[..., 1].copy()
+            mask = self.data_calculation.masks[idx_pose]
+            vals_x = self.data_calculation.x_phase_list[idx_pose]
+            vals_y = self.data_calculation.y_phase_list[idx_pose]
+
+            # Make x/y images RGB
+            im_x = im_x.astype(float) / im_x.max()
+            im_y = im_y.astype(float) / im_y.max()
+
+            im_x = np.stack([im_x] * 3, axis=2)
+            im_y = np.stack([im_y] * 3, axis=2)
+
+            # Add active pixels as colored pixels
+            cm = colormaps.get_cmap('jet')
```

Comment:
I'm not saying that you need to use this here, I just wanted to let you know about this method:

https://github.com/sandialabs/OpenCSP/blob/develop/opencsp/common/lib/render/Color.py#L212